hack/release/stable-pr.sh

```diff
@@ -1,6 +1,31 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+# updateKarpenterCoreGoMod bumps the karpenter-core go.mod to the release version so that the
+# karpenter and karpenter-core release versions match
+updateKarpenterCoreGoMod(){
+  RELEASE_VERSION=$1
+  if [[ $GITHUB_ACCOUNT != $MAIN_GITHUB_ACCOUNT ]]; then
+    echo "not updating go mod for a repo other than the main repo"
+    return
+  fi
+  go get -u "github.com/aws/karpenter-core@${RELEASE_VERSION}"
+  cd test
+  go get -u "github.com/aws/karpenter-core@${RELEASE_VERSION}"
+  cd ..
+  make tidy
+}
+
+# updateTektonPreUpgradeVersion updates the version that we use for the pre-upgrade E2E test suite
+# so that we are constantly testing against the last minor version of Karpenter
+updateTektonPreUpgradeVersion(){
+  LAST_MINOR_VERSION=$(git tag --sort=committerdate | grep -v "v${RELEASE_VERSION_MAJOR}.${RELEASE_VERSION_MINOR}" | tail -1)
```

Comment:
I don't think this works.

for example, if we release `v0.28.0` 

```
git tag --sort=committerdate | grep -v "v0.28" | tail -1
v0.28.0
```

And we want it to be `v0.27.1`. I think you'd want to subtract 1 from RELEASE_VERSION_MINOR, and then it should work.